.env

```diff
@@ -1 +1,2 @@
-GENERATE_SOURCEMAP=false
\ No newline at end of file
+GENERATE_SOURCEMAP=false
+JWT_SECRET = “BL&qc6_gwP3=??!u”
```

Comment:
uh...this can't be right, surely?

Reply:
LMAO no it's not; I'm putting .env on the gitignore. Nice catch.

Reply:
Change pushed.

Reply:
We'll have to change the SECRET now, since it's been leaked; this will be preserved in git history.

Reply:
You should also delete the tracked .env file and commit that

Reply:
Sounds good; tis done ^.^

Reply:
@QinyuanWu looks like you'll have to redistribute the new JWT_SECRET

Reply:
I think it will expire in a few weeks so i think it's fine for now lol

Reply:
In the future we'll have to xD